py/server/deephaven/table.py

```diff
@@ -70,23 +70,14 @@ def __init__(self, j_table: jpy.JType):
         self._definition = self.j_table.getDefinition()
         self._schema = None
 
-    def __repr__(self):
-        default_repr = super().__repr__()
+    def __str__(self):
+        default_str = super().__str__()
         column_dict = {col.name: col.data_type for col in self.columns[:10]}
-        repr_str = (
-            f"{default_repr[:-2]}, num_rows = {self.size}, columns = {column_dict}"
+        str_str = (
+            f"{default_str[:-2]}, num_rows = {self.size}, columns = {column_dict}"
```

Comment:
Since we talked about `__str__` -> `toString()`, does it make sense to change `Table.toString()` to output these values, instead of having this special python code?

Reply:
Probably, but in the future. I think we will want to have an even prettier version of it though, with solid expectations.